public/app/plugins/datasource/cloud-monitoring/plugin.json

```diff
@@ -7,6 +7,17 @@
   "metrics": true,
   "alerting": true,
   "annotations": true,
+  "includes": [
+    { "type": "dashboard", "name": "BigQuery Query Monitoring", "path": "dashboards/BigQuery Query Monitoring.json" },
```

Comment:
I'm wondering if it's safer to have filenames without spaces?

Reply:
@marcusolsson, are there plugin-specific recommendations? If not, should there be?

Reply:
Without spaces and lowercase is probably the safest

Reply:
thank you all for your comments; I will change them to lowercase without spaces.
I'm thinking also that `/` separator can also be an issue in Windows but I don't know how to change it to work for both. This must be [where](https://github.com/grafana/grafana/blob/master/pkg/plugins/dashboards.go#L99) it is consumed. Any ideas?

Reply:
I have tested importing dashboards in windows (from the [cloudwatch datasource](https://github.com/grafana/grafana/blob/b892aa2eb73f64f49f217f6f7c74f37cd9e73959/public/app/plugins/datasource/cloudwatch/plugin.json#L11)) and it works so the `/` separator is not a problem (as I expected).